contracts/mocks/MockOAL.sol

```diff
@@ -0,0 +1,197 @@
+// Copyright Immutable Pty Ltd 2018 - 2023
+// SPDX-License-Identifier: Apache 2.0
```

Comment:
I think this contract could just extend the OperatorAllowlist contract, just adding the extra storage location and function(s)